articles/quickstart/native/android-vnext/index.yml

```
@@ -0,0 +1,48 @@
+title: Android
+alias:
+  - Android
+language:
+  - Kotlin
+hybrid: false
+author:
+  name: Luciano Balmaceda
+  email: luciano.balmaceda@auth0.com
+  community: false
+image: /media/platforms/android.png
+topics:
+  - quickstart
+contentType: tutorial
+useCase: quickstart
+beta: true
+snippets:
+  dependencies: native-platforms/android/dependencies
+  setup: native-platforms/android/setup
+  use: native-platforms/android/use
+seo_alias: android
+default_article: 00-login
+articles:
+  - 00-login
+next_steps:
+  - path: 00-login
+    list:
+    - text: Configure other identity providers
+      icon: 345
+      href: "/identityproviders"
+    - text: Enable multifactor authentication
+      icon: 345
+      href: "/multifactor-authentication"
+    - text: Learn about attack protection
+      icon: 345
+      href: "/attack-protection"
+    - text: Learn about rules
+      icon: 345
+      href: "/rules"
+show_steps: true
+github:
+  org: auth0-samples
+  repo: auth0-android-sample
+  branch: feat-kotlin
+requirements:
+  - Android Studio 3.6.1
```

Let's use the latest just in case. Also SDK 30 and Emulator with Android 11

```suggestion
  - Android Studio 4.1.0
```